src/forms/calendarComponent.story.jsx

```diff
@@ -28,7 +25,16 @@ storiesOf('CalendarComponent', module)
 				/>
 			</Chunk>
 			<Chunk>
-				<p className="text--caption">For a full list of available <code>datepickerOptions</code>, see <a className="link" target="_blank" href="https://chmln.github.io/flatpickr/options/">flatpickr documentation</a></p>
+				<p className="text--caption">
+					For a full list of available <code>datepickerOptions</code>, see{' '}
```

Comment:
what on earth did prettier do here?

`see{' '}` <- looks like it's enforcing a space in a weird way to avoid a trailing space. 

Not a blocker, but would be nice to clean up.